docs/api.md

```diff
@@ -58,21 +58,21 @@ groups:
 
 API Group | Status | Details
 --------  | ------ | -------
-[`aws.crossplane.io/v1alpha2`] | Alpha | Core AWS resources.
-[`cache.aws.crossplane.io/v1alpha2`] | Alpha | ElastiCache managed resources and classes.
-[`compute.aws.crossplane.io/v1alpha2`] | Alpha | EKS managed resources and classes.
-[`database.aws.crossplane.io/v1alpha2`] | Alpha | RDS managed resources and classes.
-[`identity.aws.crossplane.io/v1alpha2`] | Alpha | IAM managed resources and classes.
-[`network.aws.crossplane.io/v1alpha2`] | Alpha | VPC managed resources and classes.
-[`storage.aws.crossplane.io/v1alpha2`] | Alpha | S3 managed resources and classes.
-
-[`aws.crossplane.io/v1alpha2`]: api/crossplaneio/stack-aws/aws-crossplane-io-v1alpha2.md
-[`cache.aws.crossplane.io/v1alpha2`]: api/crossplaneio/stack-aws/cache-aws-crossplane-io-v1alpha2.md
-[`compute.aws.crossplane.io/v1alpha2`]: api/crossplaneio/stack-aws/compute-aws-crossplane-io-v1alpha2.md
-[`database.aws.crossplane.io/v1alpha2`]: api/crossplaneio/stack-aws/database-aws-crossplane-io-v1alpha2.md
-[`identity.aws.crossplane.io/v1alpha2`]: api/crossplaneio/stack-aws/identity-aws-crossplane-io-v1alpha2.md
-[`network.aws.crossplane.io/v1alpha2`]: api/crossplaneio/stack-aws/network-aws-crossplane-io-v1alpha2.md
-[`storage.aws.crossplane.io/v1alpha2`]: api/crossplaneio/stack-aws/storage-aws-crossplane-io-v1alpha2.md
+[`aws.crossplane.io/v1alpha3`] | Alpha | Core AWS resources.
+[`cache.aws.crossplane.io/v1beta1`] | Beta | ElastiCache managed resources and classes.
+[`compute.aws.crossplane.io/v1alpha3`] | Alpha | EKS managed resources and classes.
+[`database.aws.crossplane.io/v1beta1`] | Beta | RDS managed resources and classes.
```

Comment:
Don't think this should necessarily look different, but just wanted to note the existence of `database.aws.crossplane.io/v1alpha3`

Reply:
yep that's a good point, the `database.aws.crossplane.io` group has 2 versions.  I chose to only list the most mature version in this table.

Reply:
Sounds reasonable to me!